TwitchLib.Api/Services/Events/LiveStreamMonitor/OnStreamMonitorTimerTickArgs.cs

```diff
@@ -1,12 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace TwitchLib.Api.Services.Events.LiveStreamMonitor
-{
-    public class OnStreamMonitorTimerTickArgs : EventArgs
-    {
-        /// <summary>Event property representing channel Id that has gone online.</summary>
-        public int CheckIntervalSeconds;
-    }
-}
```

Comment:
why are you removing this?

Reply:
Because, just like the extra properties in the other Args classes, they're redundant. You already have access to the LiveStreamMonitor's CheckIntervalInSeconds property. That's why it's removed.